src/elements/button/src/index.tsx

```diff
@@ -17,25 +18,36 @@ export const Button: React.FC<Props> = ({
   variant,
   iconPosition = null,
   onClick,
+  size = 'large',
   ...props
 }) => {
+  const { theme }: any = useThemeUI()
+
   return (
     <button
       sx={{
         cursor: 'pointer',
         backgroundImage: 'none',
         fontFamily: 'base',
-        fontSize: 'base',
-        paddingX: 'sm',
-        paddingY: 'base',
+        fontSize: theme.buttons.base.btnSize
+          ? theme.buttons.base.btnSize[size].fontSize
+          : 'base',
+        paddingX: theme.buttons.base.btnSize
+          ? theme.buttons.base.btnSize[size].paddingX
+          : 'sm',
+        paddingY: theme.buttons.base.btnSize
+          ? theme.buttons.base.btnSize[size].paddingY
+          : 'base',
```

Comment:
I _think_ this can be simplified to something like this:

```
fontSize: 'base',
paddingX: 'sm',
paddingY: 'base',
variant: `theme.buttons.base.btnSize.${size}`
```

Reply:
there is currently a variant in use on the element, and no non hacky way of solving: https://github.com/system-ui/theme-ui/issues/403 there is a fix in the works but not yet released